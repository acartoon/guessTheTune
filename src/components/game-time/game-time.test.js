import React from 'react';
import renderer from 'react-test-renderer';
// import {testMocks} from '../../mocks/test-mocks';
import GameTime from './game-time.jsx';

it(`GameTime correctly renders after relaunch`, () => {

  const tree = renderer
  .create(<GameTime
    gameTime={300}
    onStartÐ¡ountdown={() => null}
  />, {
    createNodeMock: () => {
      return {};
    }
  })
  .toJSON();
  expect(tree).toMatchSnapshot();
});
