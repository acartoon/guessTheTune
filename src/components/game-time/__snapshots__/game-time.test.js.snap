// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameTime correctly renders after relaunch 1`] = `
<div
  className="timer__value"
  xmlns="http://www.w3.org/1999/xhtml"
>
  <span
    className="timer__mins"
  >
    5
  </span>
  <span
    className="timer__dots"
  >
    :
  </span>
  <span
    className="timer__secs"
  >
    0
  </span>
</div>
`;
